modules/end-user-guide/partials/proc_mounting-a-secret-as-an-environment-variable-into-a-workspace-container.adoc

```diff
@@ -1,23 +1,61 @@
 // Module included in the following assemblies:
-//
 // mounting-a-secret-as-a-file-or-an-environment-variable-into-a-workspace-container
 
 [id="mounting-a-secret-as-an-environment-variable-into-a-workspace-container_{context}"]
 = Mounting a secret as an environment variable into a workspace container
 
-The following section describes how to mount a {platforms-name} secret from the user’s {orch-namespace} as an environment variable, or variables, into single-workspace or multiple-workspace containers of {prod-short}.
+In {prod-short}, you can mount the {platforms-name} secret from your {orch-namespace} as an environment variable (or variables) in single-workspace or multiple-workspace containers.
```

Comment:
What was wrong with the orig. version? In fact, I think it was more descriptive. The new formulation only vaguely suggests that something *can* be done. I'd just use "*This* section".

Reply:
@rkratky IBM GS, p 22: "Avoid self-referential text whenever possible, such as "This topic is about" or "This section describes". Instead, focus on the purpose or value of the content, and include words that users are likely to search for." This is a tricky topic, and I haven't come up with a standard solution for myself. Also, in a way it's something of a fluffy truism - this section is about what the content of this section is. The trick is how to phrase the introductory sentence: in conceptual modules this isn't a problem, but in procedural modules is where we start getting "You can ...".
By the way, I've already encountered complications with "This section describes": when the section gains siblings or subsections (These sections? The following sections? The sections in a different .adoc file?), this sentence becomes outdated and is less likely to be updated.
Also, if I skip this guidance, then next time around it will be raised during reviews.
@themr0c Any ideas?

Reply:
@rkratky and @themr0c :) Speaking of fluff, what else I noticed about introductory sentences to procedures is that they often contain the reused phrase from the heading, and so the same phrase appears on two adjacent lines. I don't have a solution to offer, but it does look like fluff.

Reply:
> IBM GS, p 22: "Avoid self-referential text whenever possible, such as "This topic is about" or "This section describes". Instead, focus on the purpose or value of the content, and include words that users are likely to search for."

Good point.

> in procedural modules is where we start getting "You can ...".

_You can_ in general is superfluous -- of course you can. The question is whether you are supposed to do it, or not. Our docs (not just Che) overuse _you can_.

We should describe what the procedure accomplishes. In this case, something like:

_{prod-short} allows mounting a {platforms-name} secret from the user’s {orch-namespace} as an environment variable, or variables, into single-workspace or multiple-workspace containers._

> I've already encountered complications with "This section describes": when the section gains siblings or subsections (These sections? The following sections? The sections in a different .adoc file?), the sentence becomes outdated and is less likely to be updated.

Never refer to _other_ sections. Always consider that all modules (units of content) need to make sense on their own, and the reader may not follow the doc in a linear fasion. Forf this, it's useful to use the _"Every page is page one" check. IOW, always ask yourself whether a module/section would make sense to the reader if they started reading right there.

> ntroductory sentences to procedures is that they often contain the reused phrase from the heading, and so the same phrase appears on two adjacent lines

Yeah. Ideally, the intro/context paragraph should have a bit broader description. But in a pinch, a repetition is better than not intro sentence, I'd say.

Reply:
@rkratky Thank you, Robert. This is very helpful!